src/commands/register.ts

```diff
@@ -74,21 +74,42 @@ export default class ComponentRegister extends BaseCommand {
   static args = [{
     non_sensitive: true,
     name: 'component',
-    description: 'Path to a component to register',
+    description: 'Path to a component to register. Multiple unique components are accepted. The same component register command options are applied to any and all paths provided.',
     default: './',
   }];
 
+  // overrides the oclif default parse to allow for component to be a list of components
+  async parse<F, A extends {
+    [name: string]: any;
+  }>(options?: Interfaces.Input<F>, argv = this.argv): Promise<Interfaces.ParserOutput<F, A>> {
+    if (!options) {
+      return super.parse(options, argv);
+    }
+    options.args = [];
+    for (const _ of argv) {
+      options.args.push({ name: 'filler' });
+    }
+    const parsed = await super.parse(options, argv) as Interfaces.ParserOutput<F, A>;
+    const relative_component_path_argv = (parsed.argv || []).map(argv => path.resolve(argv));
```

Comment:
[`path.resolve`](https://nodejs.org/api/path.html#pathresolvepaths) will return absolute paths, not relative paths